### DISCUSSION TITLE
hse 2.0&zfs

### DISCUSSION THREAD

**Discussion by minmfang** [hse-project/hse#47] [General]
if the filesystem,such as zfs, offer COW FEATURE, can that benefit hse block data management in hse2.0?

**Comment by smoyerx** [comment]
We have not done any testing with a COW file system.  My guess is that it would not provide any benefit, but that is speculation.

You do need to select a file system that meets the documented [requirements](https://hse-project.github.io/2.x/gs/sysreqs/).  We generally recommend XFS.
